src/api/localResolvers/contentfulCMS.js

```diff
@@ -37,15 +37,39 @@ export default context => {
 					if (!context.appConfig.contentful.enable) {
 						return {};
 					}
+
+					// setup contentful params
 					const contentfulQueryParams = {
 						content_type: contentType
 					};
 					if (contentKey) {
 						contentfulQueryParams['fields.key'] = contentKey;
 					}
 
+					// Use the contentful URL as the cache key to localStorage
+					const cacheKey = 'https://cdn.contentful.com'
+									+ `/spaces/${context.appConfig.contentful.space}`
+									+ `/environments/${context.appConfig.contentful.environment}`
+									+ `/entries?${Object.entries(contentfulQueryParams).map(entry => entry.join('=')).join('&')}`;// eslint-disable-line max-len
+
+					const cachedResponse = localStorage.getItem(cacheKey);
```

Comment:
We have [store2](https://github.com/nbubna/store) as a dependency that you can use to help avoid browser-support issues that can crop up with using localStorage directly